sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -1195,6 +1196,49 @@ def run__NativeWrite(self, transform_node, options):
          PropertyNames.STEP_NAME: input_step.proto.name,
          PropertyNames.OUTPUT_NAME: input_step.get_output(input_tag)})
 
+  @unittest.skip("This is not a test, despite the name.")
```

Comment:
```suggestion
  @unittest.skip("The test setup currently matches this method and attempts to run it as a test which is why we explicitly mark this with skip.")
```

Reply:
We opted to set __test__ = False instead of using a unittest annotation, since the annotation caused a different lint test to fail.